sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -150,15 +160,11 @@ private void processCoverage(SensorContext context, Map<File, Integer> numberOfL
     phpUnitSensor.execute(context, numberOfLinesOfCode);
   }
 
-  void analyseFiles(
-    SensorContext context,
-    PHPAnalyzer phpAnalyzer,
-    List<InputFile> inputFiles,
-    ProgressReport progressReport,
-    Map<File, Integer> numberOfLinesOfCode) {
+  void analyseFiles(SensorContext context, PHPAnalyzer phpAnalyzer, Collection<CompatibleInputFile> inputFiles, ProgressReport progressReport, Map<String, Integer> numberOfLinesOfCode) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 186 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)